Resources/Prototypes/_LostParadise/Reagents/Consumable/Drink/drinks.yml

```
@@ -0,0 +1,112 @@
+      - !type:SatiateThirst
+        factor: 2
+      - !type:AdjustReagent
+        reagent: Theobromine
+        amount: 0.05
+    Calm:
+      effects:
+      - !type:ChemAddMoodlet
+        moodPrototype: LPPCamomileTeaCalm
+
+
+- type: reagent 
```

yamllint

[error] trailing spaces

(trailing-spaces)